Content.Client/StationRecords/GeneralStationRecordConsoleWindow.xaml.cs

```diff
@@ -131,4 +184,38 @@ private void PopulateRecordContainer(GeneralStationRecord record)
             RecordContainer.AddChild(control);
         }
     }
+
+    private void FilterListingOfRecords(string text = "")
+    {
+        if (!_isPopulating)
+        {
+            OnFiltersChanged?.Invoke(_filterTypes[_currentFilterType], text);
+        }
+    }
+
+    private string GetTypeFilterLocals(int id, bool isForButton = true)
+    {
+        string type = "";
+        switch (_filterTypes[id])
+        {
+            case GeneralStationRecordFilterType.Name:
+                type = "name";
+                break;
+            case GeneralStationRecordFilterType.Prints:
+                type = "prints";
+                break;
+            case GeneralStationRecordFilterType.DNA:
+                type = "dna";
+                break;
+        }
+
+        if (isForButton)
+        {
+            return Loc.GetString($"general-station-record-{type}-filter");
```

Comment:
similar thing as the other review where you can just iterate through enum, when iterating through all enum values above you can just use `.ToString()` and then lowercase it with `.ToLower()`